### PR TITLE
Ability to learn without and predict wih missing values

### PR DESCRIPTION
**Pull request by markotoplak**
All data sets need preprocessors for testing as we can always have unknown values in testing data even when there are none in learning data.

### CODE REVIEW

**Comment by markotoplak** [comment]
@lanzagar @VesnaT Anže, prosim poglej tole.

**Comment by VesnaT** [comment]
Please rebase to the latest master.

File: Orange/tests/test_classification.py [hunk 1]
```diff
@@ -209,6 +210,13 @@ def test_continuous(lf):
         assert(all(tree(strlist) == tree(Orange.data.Table(table.domain, strlist))))
 
 
+class UnknownValuesInPrediction(unittest.TestCase):
+    def test_unknown(self):
+        table = Table("iris")
+        tree = Orange.classification.TreeLearner()(table)
+        tree([1, 2, None])
```
Comment (VesnaT) [inline Orange/tests/test_classification.py]: This test is pointless - it passes without changes. Maybe you could replace TreeLearner with LogisticRegressionLearner.
